### PR TITLE
mon,mgr: move reweight-by-* to mgr

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

File: src/mgr/DaemonServer.cc [hunk 1]
```diff
@@ -557,27 +548,29 @@ bool DaemonServer::handle_command(MCommand *m)
...
 
-  std::string rs;
-  rs = ss.str();
-  dout(1) << "do_command r=" << r << " " << rs << dendl;
-  if (con) {
-    MCommandReply *reply = new MCommandReply(r, rs);
-    reply->set_tid(m->get_tid());
-    reply->set_data(odata);
-    con->send_message(reply);
+bool DaemonServer::_reply(MCommand* m,
```
Comment (tchaikov) [inline src/mgr/DaemonServer.cc]: will remove the always-return bool logic in another PR.

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
retest this please.

**Comment by liewegas** [comment]
Error EDOM: Refusing to reweight: we only have 27 PGs across 3 osds!

/a/sage-2017-04-11_21:07:54-rados-wip-sage-testing---basic-smithi/1013639

**Comment by tchaikov** [comment]
tested at http://pulpito.ceph.com/kchai-2017-04-13_05:10:51-rados-wip-sage-testing---basic-smithi/

**Pull request merged**